test/controllers/sessions_controller_test.rb

```diff
@@ -15,6 +15,14 @@ def test_login_succeeds_with_whitespaced_valid_account_number
     assert_redirected_to(:articles)
   end
 
+  def test_login_succeeds_with_valid_account_number_and_empty_return_to_param
+    user = User.new(account_number: '1234123412341234')
+    user.save
+    post(login_url, params: { return_to: '  ', account_number: user.account_number })
+
+    assert_redirected_to(:articles)
```

Comment:
Without this pull request, this assertion would fail because the redirect would lead to the root of the app (`/`) because `params[:return_to]` is sometimes an empty string.